### PR TITLE
Remove help us section

### PR DESCRIPTION
**Pull request by anselmbradford**
Removes help us section and anything that this field should be used for should go into GH issues in this repo.

## Removals

- Removes help us section.

## Testing

- Visit preview link in this PR and go to the footer of a page and hit the GH issue link.

### CODE REVIEW

**Comment by anselmbradford** [comment]
Adds this to the footer:

Screen Shot 2020-05-06 at 1 54 20 PM

File: docs/_includes/footer.html [hunk 1]
```diff
@@ -5,12 +5,23 @@
...
-                <time datetime="{{ page.last_modified_at }}">
-                    {{ page.last_modified_at | date: "%l:%M:%S %p %b %d, %Y" }}
-                </time>
-            </small>
+            <p>
+                <small>
+                    <strong>Help improve this page:</strong>
+                    <a href="https://github.com/cfpb/design-system/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
+                        File an issue on GitHub
+                    </a>.
```
Comment (contolini) [inline docs/_includes/footer.html:13]: This is putting a space before the period.

Screen Shot 2020-05-07 at 1 06 48 PM

File: docs/_includes/footer.html [hunk 2]
```diff
@@ -5,12 +5,23 @@
...
-            <small>
-                <strong>Page last edited:</strong>
-                <time datetime="{{ page.last_modified_at }}">
-                    {{ page.last_modified_at | date: "%l:%M:%S %p %b %d, %Y" }}
-                </time>
-            </small>
+            <p>
+                <small>
+                    <strong>Help improve this page:</strong>
+                    <a href="https://github.com/cfpb/design-system/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
```
Comment (contolini) [inline docs/_includes/footer.html:11]: Keep it repo agnostic:

```suggestion
                    <a href="https://github.com/{{ site.repository }}/issues/new?assignees=&labels=content%2C+help+wanted&template=component-needs-content-edits.md&title=Suggestions+for%3A+%22{{ page.title }}%22">
```

**Comment by anselmbradford** [comment]
> bar-charts.md and introductions.md also have help us sections to remove.

Erg I had them both edited in my editor but forgot to save :(

File: docs/_includes/footer.html (hunk 1 above)
Comment (anselmbradford) [reply docs/_includes/footer.html:13]: Dang it. Maybe I'll include the period in the link.

**Pull request merged**